qa/common/gen_qa_models.py

```diff
@@ -1672,6 +1913,10 @@ def create_fixed_models(models_dir,
                         required=False,
                         action='store_true',
                         help='Generate Pytorch LibTorch models')
+    parser.add_argument('--python',
+                        required=False,
+                        action='store_true',
+                        help='Generate Python models')
```

Comment:
Python (PyTorch) models? Since currently you are only generating pytorch models

Reply:
In the new version, they are Python models.